backend/src/test/java/club/devcord/devmarkt/Seed.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2022 Contributors to the Devmarkt-Backend project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package club.devcord.devmarkt;
+
+import club.devcord.devmarkt.entities.template.Question;
+import club.devcord.devmarkt.entities.template.Template;
+import java.util.List;
+import java.util.Map;
+
+public class Seed {
+
+  public static final Map<String, Template> SEED = Map.of(
```

Comment:
What about `TEMPLATE_SEED`? When the applications or permissions come into the seeder, I think we can't just throw them in here because of the types